.github/workflows/deploy.yml

```diff
@@ -35,20 +35,21 @@ jobs:
     - name: Deploy to cloud.gov
       uses: 18f/cg-deploy-action@main
       env:
-        DANGEROUS_SALT: ${{ secrets.DANGEROUS_SALT }}
-        SECRET_KEY: ${{ secrets.SECRET_KEY }}
+        DANGEROUS_SALT: ${{ secrets.PROD_DANGEROUS_SALT }}
+        SECRET_KEY: ${{ secrets.PROD_SECRET_KEY }}
+        ADMIN_CLIENT_SECRET: ${{ secrets.PROD_ADMIN_CLIENT_SECRET }}
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       with:
         cf_username: ${{ secrets.cloudgov_username }}
         cf_password: ${{ secrets.cloudgov_password }}
         cf_org: gsa-10x-prototyping
         cf_space: 10x-notifications
-        full_command: |
-          cf push --strategy rolling \
```

Comment:
Removing rolling strategy permanently, or just to coordinate the secrets update?

Reply:
It's still using rolling, that's the default for the action code when calling with just `push_arguments` rather than the full command.